src/lib.rs

```diff
@@ -405,6 +405,84 @@ impl BBox {
     }
 }
 
+#[derive(Clone, Copy, PartialEq)]
+pub struct Transform {
+    pub a: f32,
+    pub b: f32,
+    pub c: f32,
+    pub d: f32,
+    pub e: f32,
+    pub f: f32,
+}
+
+impl Transform {
+    #[inline]
+    pub fn new(a: f32, b: f32, c: f32, d: f32, e: f32, f: f32) -> Self {
+        Transform { a, b, c, d, e, f }
+    }
+
+    #[cfg(feature = "variable-fonts")]
```

Comment:
this cfg is a bit strange

Reply:
Yes, we should either make it private, as it was before, or remove cfg altogether.